### PR TITLE
Change error message color

### PR DESCRIPTION
**Pull request by CindyTsai1**
Enhancement on UI

### CODE REVIEW

File: src/main/java/seedu/address/ui/ResultDisplay.java [hunk 1]
```diff
@@ -36,6 +37,18 @@ public ResultDisplay() {
     private void handleNewResultAvailableEvent(NewResultAvailableEvent event) {
         logger.info(LogsCenter.getEventHandlingLogMessage(event));
         Platform.runLater(() -> displayed.setValue(event.message));
+
+        if(event.isError) setErrorStyle();
```
Comment (nbriannl) [inline src/main/java/seedu/address/ui/ResultDisplay.java]: Checkstyle violation reported correctly. Did you configure your checkstyle correctly?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -361,6 +399,20 @@ Clears all entries from the address book. +
 Format: `clear` +
 Alternatives: `c`, `wipe`
 
+=== Setting a password for your address book : `password` _(Coming in v2.0)_
+
+Sets or changes the password that allows access to the address book +
+Format: `password PASSWORD` +
+Alternatives: `p` , `code`
```
Comment (April0616) [inline docs/UserGuide.adoc]: how to cancel the password?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -56,24 +56,41 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
+Adds a person to address book from the social media accounts +
+Format: `add s/SOCIAL_MEDIA_TYPE SOCIAL_MEDIA_ID`
+
+****
+* The name of social media type is case insensitive.
+****
+
+Examples:
+
+* `add s/facebook John Doe`
```
Comment (April0616) [inline docs/UserGuide.adoc]: maybe give more examples?

**Review by April0616 — DISMISSED** [review]
Generally good! Maybe you should check the travis ci and make some changes

**Review by nbriannl — APPROVED** [review]
On top of Yuting's review (which have been addressed), all is good.

**Pull request merged**